translations/base-fr.yaml

```diff
@@ -268,14 +268,14 @@ ingame:
         title: Statistiques
         dataSources:
             stored:
-                title: Stocké
+                title: Stock
```

Comment:
Yes, but It makes more sense to have it be this way I think because it shows the stock you have right now and not the all-time stock. Also for the others it just sounds better and same, it is showing what is happening rn